.github/scripts/get_tutorials_stats.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
+import os.path
+from typing import List, Optional, Tuple, Dict, Union
+import boto3  # type: ignore[import]
+from botocore.exceptions import ClientError  # type: ignore[import]
+from pprint import pprint
+
+dynamodb = boto3.resource('dynamodb', region_name="us-east-1")
+
+def run_command(cmd: str, cwd: Optional[str] = None) -> str:
+    """
+    Run a shell command.
+
+    Args:
+        cmd: Command to run
+        cwd: Working directory
+    Returns:
+        Output of the command.
+    """
+    import shlex
+    from subprocess import check_output
```

Comment:
Prob best to move these to the top of the file so that they don't have to be imported every time this function is invoked

Reply:
moved